src/utils/feedbackFormURL.ts

```diff
@@ -0,0 +1,16 @@
+export const feedbackFormURL = (
+  referrer: 'deploy' | 'stoned-banny',
```

Comment:
**praise**: great utility function!

**suggestion (non-blocking)**: with utils like these it can sometimes be nicer to define a single object as the parameter. That way when a developer uses the function, they can see at a glance what a given parameter is doing.

This:
```js
feedbackFormURL('stoned-banny', 'johnnyDAO', '0x00')
```
vs this
```js
feedbackFormURL({
  referrer: 'stoned-banny',
  projectHandle: 'johnnyDAO',
  userAddress, '0x00',
})
```

Reply:
Ahh yep good tip. I'd make a note of that from now on